dds/idl/metaclass_generator.h

```diff
@@ -9,6 +9,7 @@
 #define metaclass_generator_H
 
 #include "dds_generator.h"
+#include <string>
```

Comment:
```suggestion

#include <string>
```

Reply:
This might have been used at some point when I was working but I don't remember where. I will just remove it.